README.md

```diff
@@ -11,18 +11,31 @@ to perform various operations on them.
 
 ## Perform Operations on Arrays
 
-We're going to write code to get all of the following results from the integers
-in our array:
+Your task is to write a series of methods that extract specific values from an
```

Comment:
Is this language a little too technical for the audience? Might be worthwhile to note that integers are whole numbers and use different language (or expand on) "input array" so they don't get confused.

Reply:
agreed